LayoutTests/fast/canvas/canvas-createPattern-video-loading.html

```diff
@@ -32,13 +33,13 @@
     document.body.appendChild(canvases);
 
     var video = document.createElement("video");
-    video.addEventListener("loadeddata", loadeddata);
-    video.addEventListener("playing", playing);
 
     shouldBeNull("document.createElement('canvas').getContext('2d').createPattern(video, 'repeat')");
 
     video.src = findMediaFile("video", "../../media/content/test");
 
+    Promise.all([ once(video, "loadeddata"), waitForVideoFrame(video) ]).then(firstframe);
```

Comment:
Do we even needed to wait for loadeddata event?

Reply:
probably not, but it was used before, and for some reasons I was getting failure without it at first.